profile-service/src/main/resources/db/migration/V9.16__add_tss_support_admin_role.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO `Role` (`name`)
+VALUES ('TssSupportAdmin')
```

Comment:
There are two general conventions for role names, which do we want to follow?
 1. `TssSupportAdmin`
 2. `TSS Support Admin`

Reply:
Yeah, I found there's almost half-half type of conventions for the existing roles in the table. It seems tend to be named without space from the most recent ones. However, if it is not the convention we have to follow, I feel `TSS Support Admin` looks clearer to me. But it is only my thought. What would you prefer?

Reply:
While we intend to hide this from UM, if we did want it visible in the future `TSS Support Admin` looks better there.

Reply:
Yep, that makes sense. Will update the role name as `TSS Support Admin`. Thanks @Judge40 .